view/theme/quattro/dark/style.css

```diff
@@ -1210,7 +1210,7 @@ section {
 .wall-item-container .wall-item-actions-social a {
   margin-right: 3em;
 }
-.wall-item-container .wall-item-actions-social a.active {
+.wall-item-container .wall-item-actions-social a .active {
```

Comment:
This is an error in the quattro.less file. At https://github.com/tobiasd/friendica/blob/e223fee40d349cb8080a6b2ec5000f163a3e1dda/view/theme/quattro/quattro.less#L579

It should read `&.active { font-weight: bold;}` instead. Can you please fix it and run lessc again please?